src/tilemaker.cpp

```diff
@@ -163,6 +176,15 @@ int main(int argc, char* argv[]) {
 	if (vm.count("output")==0) { cerr << "You must specify an output file or directory. Run with --help to find out more." << endl; return -1; }
 	if (vm.count("input")==0) { cout << "No source .osm.pbf file supplied" << endl; }
 
+	vector<string> bboxElements;
```

Comment:
put this inside a function (parsebbox?), which returns std::vector<double>, and uses the lexical_cast. 
Because the main is already quite large